CourseGrab/GoogleService-Info.plist

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We should add this file to the `.gitignore` so just add `CourseGrab/GoogleService-Info.plist`.

Reply:
Is it even possible to take it down from GitHub or is it going to be in version control permanently? Is that a problem?

Reply:
yeah you should be able to do so by:
1) deleting the file and make a new commit
2) pushing the commit
3) add file back to project as well as to gitignore
4) make new commit
5) squash the two previous commits into one

Reply:
let me know if you need help doing this

Reply:
Should be fine once you squash. Or u can just amend the commit. But yea just add to gitignore and make sure to remove this file from repo

Reply:
Should be fixed